_calls/fields-search.md

```diff
@@ -0,0 +1,532 @@
+---
+layout: call
+priority: 4
+title:  Recherche personnalisée
+description: > #
+ Le variable <b>fields</b> vous permet de filter les informations.
+
+ <h3 class="ui header">Champs possibles:</h3>
+ <ul class="ui list">
+   <li>code</li>
+   <li>nom</li>
+   <li>codesPostaux</li>
+   <li>surface</li>
+   <li>population</li>
+   <li>centre</li>
+ </ul>
```

Comment:
Yes I do second that. You should put the expected json result in `yaml` in the header and use [`jsonify`](https://jekyllrb.com/docs/templates/#filters)